src/Formulae.jl

```diff
@@ -124,3 +124,15 @@ end
 function apply_synergy_formula(interaction_information, redundancy)
 	return interaction_information + redundancy
 end
+
+# Parameters:
+#	- probabilities of first variable
+#	- probabilities of second variable
+#	- base of exponent
+function apply_cross_entropy_formula{T<:AbstractFloat, R<:Real}(
```

Comment:
Could you line-break this similarly to the above functions? (Just for internal consistency - could think harder about line length in the future.)